inz-personal-data/Issues/personName/description.xml

```diff
@@ -0,0 +1,38 @@
+<IssueSpecification version="4">
+    <Name>
+        <en>Demo INZ Personal Data Person Name</en>
+        <nl>Demo INZ Personal Data Person Name</nl>
```

Comment:
This too I would recommend merging with the other two credentials of this issuer; compare with [this production credential](https://github.com/privacybydesign/pbdf-schememanager/blob/master/gemeente/Issues/personalData/description.xml) which also contains all of this data in one credential.